docs/motivation.md

```diff
@@ -0,0 +1,91 @@
+effort.**
+
+Compilation is an incredibly important component of quantum computing infrastructure. The inability
+to optimize large algorithms for future fault-tolerant hardware will extend (or block!) timelines
+for utility-scale quantum computation.
+
+However, there is still a lot of work to do, and a lot of uncertainty. We need to:
+
+- continue to make new theoretical compilation breakthroughs; 
+- identify the most-promising large scale algorithms; and finally,
+- figure out how to make this practical through quantum software.
+  
+To do so, we need to bridge the gap between researchers doing fundamental compilation and
+algorithmic research, and quantum software developers.
+
+- **How can we share results and ideas faster, within a common framework?**
+- **How do we make sure we are speaking a common language, and aware of challenges and approaches
+    on both sides?**
+
+## Thinking beyond circuits.
+
+**Quantum algorithms have structure that we need to preserve for scalability, but thinking in
+circuits is holding us back**.
+
+This is already recognized and built into common quantum software
+libraries (Qiskit, CUDA-Q, PennyLane, UCC, Guppy, TKET, etc.) and formats (QIR, OpenQASM3, etc.),
+but introduces complexity into the compilation pipeline.
+
+- While we are starting to see [exceptions](https://arxiv.org/abs/2410.23493), most theoretical
+  compilation work is still assuming that compilation passes are applied to a straight line list
+  of quantum instructions, and implemented using formats such as OpenQASM2. It can then be highly
+  non-trivial to then map such passes to apply to structured quantum programs. How do we
+  encourage the field as a whole to consider compiling structured programs *by default*?
+
+- There is tension between maximal optimization and compact program representation. There are
+  still a lot of questions we need to consider and resolve when it comes to compiling dynamic,
+  structured programs. For example, the more compact the program representation (which is great
+  for compilation scalability!) the less information we potentially have up-front to maximally
+  optimize programs at compile time. How do we deal with such scenarios?
+
+## Bridging the software divide.
+
+**To solve the above problems, we need better interoperability and visibility between compilers**.
+
+There are a wealth of quantum software compilers currently available, but they are not
+interoperable, and it is often hard to inspect/debug to see how your program changes. Furthermore,
+when *implementing* your own compilation pass, you often need to choose which framework to support,
+making it harder to share ideas and knowledge.
+
+Note that some formats have become de-facto exchange formats for compilers (such as OpenQASM), but
+were not designed to be exchange formats for compilers, and as such suffer from quantum
+representations that are not ideal for high-level optimization, lucklustre tooling, and
```

Comment:
```suggestion
representations that are not ideal for high-level optimization, lacklustre tooling, and
```

Reply:
thanks, I did stare at this word for a while wondering if something wasn't quite right